pkgs/applications/window-managers/windowchef/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, libxcb, libXrandr
+, xcbutil, xcbutilkeysyms, xcbutilwm, xcbproto
+}:
+
+stdenv.mkDerivation rec {
+  pname = "windowchef";
+  version = "v0.5.0";
```

```suggestion
  version = "0.5.0";
```

We prefer not to use letters in versions.

---

This is actually a unique circumstance because while I was creating this package I was added as a maintainer on commit #84284.  However, when I looked it was bhougland instead of bhougland18, so i acually removed myself from the maintainers on the third commit.  It was a crazy timing issue.